src/chainparams.h

```diff
@@ -17,6 +17,8 @@ using namespace std;
 #define MESSAGE_START_SIZE 4
 typedef unsigned char MessageStartChars[MESSAGE_START_SIZE];
 
+inline bool TestNet();
```

Comment:
@tondaatouk does this properly reference the TestNet variable that is available globally in most other page(s)?